File: common/src/main/java/com/ssblur/scriptor/item/ScriptorTags.java
package com.ssblur.scriptor.item;

import com.ssblur.scriptor.ScriptorMod;
import net.minecraft.core.registries.Registries;
import net.minecraft.tags.TagKey;
import net.minecraft.world.item.Item;

public class ScriptorTags {
  public static final TagKey<Item> READABLE_SPELLBOOKS = TagKey.create(Registries.ITEM, ScriptorMod.location("readable_spellbooks"));
}

File: common/src/main/kotlin/com/ssblur/scriptor/item/ScriptorTags.kt
package com.ssblur.scriptor.item

import com.ssblur.scriptor.ScriptorMod
import net.minecraft.tags.TagKey
import net.minecraft.world.item.Item

object ScriptorTags {
  val READABLE_SPELLBOOKS: TagKey<Item> = ScriptorMod.registerItemTag("readable_spellbooks")
  val WRITABLE_SPELLBOOKS: TagKey<Item> = ScriptorMod.registerItemTag("writable_spellbooks")
  val SPELLBOOKS: TagKey<Item> = ScriptorMod.registerItemTag("spellbooks")
}